site/themes/docsy/layouts/partials/head.html

```diff
@@ -14,7 +14,6 @@
 {{- template "_internal/opengraph.html" . -}}
 {{- template "_internal/google_news.html" . -}}
 {{- template "_internal/schema.html" . -}}
-{{- template "_internal/twitter_cards.html" . -}}
```

Comment:
Ah nevermind, does this take care of the default tags?

Reply:
yes

Reply:
btw, PTAL @#335 when it is deployed